Calibration/HcalIsolatedTrackReco/src/EcalIsolatedParticleCandidateProducer.cc

```diff
@@ -211,8 +211,8 @@ EcalIsolatedParticleCandidateProducer::produce(edm::Event& iEvent, const edm::Ev
   // is put into the Event
   
 //  std::cout<<"put cand into event"<<std::endl;
-  std::auto_ptr<reco::IsolatedPixelTrackCandidateCollection> pOut(iptcCollection);
-  iEvent.put(pOut);
+  std::unique_ptr<reco::IsolatedPixelTrackCandidateCollection> pOut(iptcCollection);
+  iEvent.put(std::move(pOut));
 
```

Comment:
I agree - but doesn't have anything to do with the current PR. You might feedback to the hcal group?